src/data_processing/loading_tokenization.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import pandas as pd
+import os
+from transformers import GPT2Tokenizer
+import torch
+from src.data_processing.Formality_Transfer_Dataset import FormalityTransferDataset
+import pickle
+
+# Defining constants
+# Parameters
+MODEL_CHECKPOINT = 'gpt2-medium'
+PREFIX_FORMAL = '<|formal|>'
+PREFIX_INFORMAL = '<|informal|>'
+MAX_INPUT_LEN = 2048
+MAX_TARGET_LEN = 128
+
+
+# Establishing the directory where the script is executed and constructing the path of the dataset
+def establish_data_path():
+    path = os.getcwd()
+    path = os.path.abspath(os.path.join(path, os.pardir))
+    path = os.path.abspath(os.path.join(path, os.pardir))
+    path = os.path.join(path, 'data/raw/GYAFC_Corpus')
```

Comment:
The two `path = os.path.abspath(os.path.join(path, os.pardir))` are only necessary if you have cd into this folder when running the cell. Either mention that you have to cd in the folder to run this file or take this two lines out

Reply:
I ll take them out, that makes sense